crates/telio-model/src/api_config.rs

```diff
@@ -189,14 +189,18 @@ pub struct FeatureDirect {
     /// Configuration options for skipping unresponsive peers
     #[serde(default = "FeatureDirect::default_skip_unresponsive_peers")]
     pub skip_unresponsive_peers: Option<FeatureSkipUnresponsivePeers>,
+    /// Parameters to optimize battery lifetime
+    #[serde(default)]
+    pub battery_optimization: Option<FeatureBatteryOptimization>,
```

Comment:
Could we name this value slightly more specific? As it will be in a somewhat "public" interface.

For example, `endpoint_providers_usage_optimization`.

Reply:
renamed to endpoint_providers_optimization